uds.js

```diff
@@ -155,293 +155,295 @@
+    uds.postRoleLevel = function (userId, roleName, roleLevel) {
+        var url = udsHostName.clone().setPath('/user/' + userId + "/role-level/" + roleName);
+        return executeUdsAjaxCallWithData(url, roleLevel, 'PUT');
     };
-    
-    uds.postEditPrivateComments = function (caseNumber,caseComment,caseCommentId,draft) {
-        var url = udsHostName.clone().setPath('/case/' + caseNumber + "/comments/"+ caseCommentId+"/private");
+
+    uds.postEditPrivateComments = function (caseNumber, caseComment, caseCommentId, draft) {
+        var url = udsHostName.clone().setPath('/case/' + caseNumber + "/comments/" + caseCommentId + "/private");
         url.addQueryParam('draft', draft);
-        return executeUdsAjaxCallWithData(url,caseComment,'PUT');
+        return executeUdsAjaxCallWithData(url, caseComment, 'PUT');
     };
 
-    uds.createCaseNep = function(caseNumber, nep) {
+    uds.createCaseNep = function (caseNumber, nep) {
         var url = udsHostName.clone().setPath('/case/' + caseNumber + "/nep");
-        return executeUdsAjaxCallWithData( url, nep,'POST');
+        return executeUdsAjaxCallWithData(url, nep, 'POST');
     };
 
-    uds.updateCaseNep = function(caseNumber, nep) {
+    uds.updateCaseNep = function (caseNumber, nep) {
         var url = udsHostName.clone().setPath('/case/' + caseNumber + "/nep");
-        return executeUdsAjaxCallWithData( url, nep,'PUT');
+        return executeUdsAjaxCallWithData(url, nep, 'PUT');
     };
 
-    uds.removeCaseNep = function(caseNumber) {
+    uds.removeCaseNep = function (caseNumber) {
         var url = udsHostName.clone().setPath('/case/' + caseNumber + "/nep");
-        return executeUdsAjaxCall( url ,'DELETE');
+        return executeUdsAjaxCall(url, 'DELETE');
     };
 
-    uds.getAvgCSATForAccount = function(uql) {
-        var url = udsHostName.clone().setPath('/metrics/CsatAccountAvg').addQueryParam('where', encodeURIComponent(uql));
-        return executeUdsAjaxCall(url,'GET');
+    uds.getAvgCSATForAccount = function (uql) {
+        var url = udsHostName.clone().setPath('/metrics/CsatAccountAvg').addQueryParam('where', uql);
+        return executeUdsAjaxCall(url, 'GET');
     };
 
-    uds.getCaseContactsForAccount = function(accountNumber){
+    uds.getCaseContactsForAccount = function (accountNumber) {
         var url = udsHostName.clone().setPath('/account/' + accountNumber + "/contacts");
-        return executeUdsAjaxCall(url,'GET');
+        return executeUdsAjaxCall(url, 'GET');
     };
 
-    uds.getCaseGroupsForContact = function(contactSSO){
+    uds.getCaseGroupsForContact = function (contactSSO) {
         var url = udsHostName.clone().setPath('/case/casegroups/user/' + contactSSO);
-        return executeUdsAjaxCall(url,'GET');
+        return executeUdsAjaxCall(url, 'GET');
     };
 
-    uds.getRMECountForAccount = function(uql){
-        var url = udsHostName.clone().setPath('/case/history').addQueryParam('where', encodeURIComponent(uql));
```

Comment:
It is no longer required with jsuri 1.3.1.  I believe that the latest version of jsuri auto-escapes query params.  If we encode those query params then the result is double encoded, which breaks things.